src/compile/data/nonpositivenullfilter.ts

```diff
@@ -14,7 +13,7 @@ export namespace nonPositiveFilter {
   export function parseUnit(model: Model): Dict<boolean> {
     return model.channels().reduce(function(nonPositiveComponent, channel) {
       const scale = model.scale(channel);
-      if (!model.field(channel) || !scale) {
+      if (!model.has(channel) || !scale) {
```

Comment:
this line is very interesting.  I don't know why it was `model.field`-- any idea?